src/viewProviders/resources.ts

```diff
@@ -428,10 +454,16 @@ export class LocalConnectionRow extends SingleEnvironmentConnectionRow<
   }
 
   get status(): string {
-    const isDockerAvailable = getContextValue(ContextValues.dockerServiceAvailable) === true;
+    const isDockerAvailable =
+      getContextValue<boolean>(ContextValues.dockerServiceAvailable) === true;
 
     if (isDockerAvailable) {
-      return this.connected ? this.kafkaCluster!.uri! : "(Local Kafka not running)";
+      if (this.kafkaCluster) {
+        return this.kafkaCluster.uri!;
+      } else if (this.medusa) {
+        return "Only Medusa available";
+      }
+      return "(No connection)";
```

Comment:
Future: adjust this since it reverts a previous change going from `(No connection)` to `(Local Kafka not running)`